release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -567,6 +567,29 @@ For more information, see xref:../openshift_images/image-configuration.adoc#imag
 [id="ocp-4-8-monitoring"]
 === Monitoring
 
+[id="ocp-4-8-monitoring-alerting-rule-changes"]
+==== Alerting rule changes
+
+{product-title} 4.8 includes the following alerting rule changes:
+
+.Alerting rule changes
```

Comment:
This renders as "Example 1. Alerting rule changes" - maybe just a `=====` heading would work?

Reply:
@jeana-redhat Hmm, in my local preview, it renders as a drop-down for collapsible content. Not sure why it isn't in the netlify preview.

Reply:
Ah, interesting. I haven't had call for using the collapsible format yet, so I didn't realize it also uses the `.` prefix. Good luck puzzling this one out! :thinking: :female_detective: